src/app/csvparser.js

```diff
@@ -1,20 +1,24 @@
+/**
+ * You need to rename this file to CsvParser.js
+ * Good OS are case-sensitive. :-)
```

Comment:
You don't have to capitalize the files if you don't what. 
But your require("somename") needs to match the case of your file.